uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -0,0 +1,695 @@
+/* eslint-env amd, jasmine */
+
+(function (CRM) {
+  define([
+    'common/lodash',
+    'common/moment',
+    'mocks/helpers/helper',
+    'common/mocks/data/contact.data',
+    'mocks/data/absence-period-data',
+    'mocks/data/absence-type-data',
+    'mocks/data/leave-request-data',
+    'mocks/data/option-group-mock-data',
+    'mocks/data/public-holiday-data',
+    'mocks/data/work-pattern-data',
+    'mocks/apis/leave-request-api-mock',
+    'mocks/apis/option-group-api-mock',
+    'mocks/apis/work-pattern-api-mock',
+    'leave-absences/my-leave/app'
+  ], function (_, moment, helper, ContactData, AbsencePeriodData, AbsenceTypeData, LeaveRequestData, OptionGroupData, PublicHolidayData, WorkPatternData) {
+    'use strict';
+
+    describe('leaveCalendarMonth', function () {
+      var $componentController, $log, $provide, $q, $rootScope, Calendar,
+        LeaveRequest, OptionGroup, controller, daysInFebruary, february, leaveRequestInFebruary,
+        period2016, publicHolidays;
+      var currentContactId = CRM.vars.leaveAndAbsences.contactId;
```

Comment:
It should be assigned in globals.js, yes. But I don't see a point of use in tests.